medusa/logger/__init__.py

```diff
@@ -346,7 +346,10 @@ def key(self):
     @property
     def issue_title(self):
         """Return the expected issue title for this logline."""
-        result = self.traceback_lines[-1] if self.traceback_lines else self.message
+        if self.traceback_lines:
+            result = next((line for line in reversed(self.traceback_lines) if line.strip()), self.message)
```

Comment:
Is the `if` statement really needed?

Reply:
Yes. Traceback lines can have an empty line, causing issue titles like this:
https://github.com/pymedusa/Medusa/issues/4369